extensions/positron-python/package.json

```diff
@@ -556,7 +556,7 @@
                 "python.defaultInterpreterPath": {
                     "default": "python",
                     "markdownDescription": "%python.defaultInterpreterPath.description%",
-                    "scope": "machine-overridable",
+                    "scope": "resource",
                     "type": "string",
                     "tags": [
                         "interpreterSettings"
```

Comment:
Just a note that we have agreed to change the settings scope of `python.defaultInterpreterPath` to "resource", deviating from upstream. We believe this is reasonable because the scope is being expanded to allow for this setting to be changed at the User Settings level in Positron on Workbench. The existing ways to modify this setting are preserved.